scripts/onload.js

```diff
@@ -37,18 +37,6 @@ var loadedPolygonLayers = [];
 
 
 
-map.on('click', 'school_house_senate_districts_UNION-poly', function (e) {
-	var pointFeatures = map.queryRenderedFeatures(e.point, {layers: loadedPointLayerNames});
-	if (pointFeatures.length === 0) {
-		popup = new mapboxgl.Popup()
-			.setLngLat(e.lngLat)
-			.setHTML(fillpopup(e.features[0].properties))
-			.addTo(map);
-	}
-	popupYear = 0;
-});
```

Comment:
This on click handler was supposed to show the district name, but it's currently broken for ESCs because we don't have those in the data.  It did still work for ISCs, but for ESCs it was giving us empty popups.  I could make it work for ESCs basically using the mirror of the LUT from yesterday, but that would be another 1-2 hours' work and I'm not sure it's worth it.  Let me know, and I can do it next week if it seems worthwhile.

Reply:
Honestly I don't think it's worth it, but let's see what CBF says. If the user is going to choose from the dropdown, they'll see what they've chosen, and it will also appear in the stats box.

Reply:
Yeah, that makes sense.  I think it was more relevant when people could zoom to a school district but have legislative district boundaries showing, and now the map's a little clearer in itself.